src/pe_reports/data/db_query.py

```diff
@@ -0,0 +1,193 @@
+"""Query the PE postgres database."""
```

Comment:
Minor I know, but we are trying to be more exacting with our comments.  There are a few of these- feel free to batch them up in a single commit.
```suggestion
"""Query the PE PostgreSQL database."""
```